### ISSUE TITLE
CKV_AZURE_11 not accepting configuration for "Allow access to Azure services"

### ISSUE THREAD

**Issue by SebastianBalle** [bridgecrewio/checkov#1144]
**Describe the bug**
The rule should accept the configuration to "Allow access to Azure services" described [here](https://docs.microsoft.com/en-us/rest/api/sql/2020-11-01-preview/firewallrules/createorupdate). This rule is critical to have in a pre-commit hook, however, must be ignored for now to allow for this configuration. 

**To Reproduce**
Define the following resource
```hcl
resource "azurerm_postgresql_firewall_rule" "allow_azure_services" {

  # General
  name                = "allow-azure-services"
  resource_group_name = azurerm_resource_group.primary.name

  # Server
  server_name = azurerm_postgresql_server.main.name

  # IP
  start_ip_address = "0.0.0.0"
  end_ip_address   = "0.0.0.0"

}
```

**Expected behavior**
The rule should accept the configuration as both IP addresses have been set to 0.0.0.0. 

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 LTS
 - Checkov Version 2.0.86

**Additional context**
Add any other context about the problem here (e.g. code snippets).

**Comment by gruebel** [comment]
@schosterbarak can you close this issue, it was fixed with PR #1134

**Comment by SebastianBalle** [comment]
I will close the issue.
